### PR TITLE
Squash users migrations and create management command for project page announcement

### PR DESCRIPTION
**Pull request by jsnshrmn**
[//]: # (Thank you for uploading a PR to the Wikipedia Library!)

## Description
- Manipulate `users` and `resources` migration graph to allow squashing and create squash migrations
- Creates a management command, email task, and email templates for project page announcement.

## Rationale
- To actually complete [T255995](https://phabricator.wikimedia.org/T255995) and [T272367](https://phabricator.wikimedia.org/T272367).
- We want to invite active editors to participate in our upcoming projects for the next two quarters.
- Note that there will be a followup push to drop the redundant migrations after the squash has deployed to servers.

## Phabricator Ticket
https://phabricator.wikimedia.org/T255995
https://phabricator.wikimedia.org/T272367

## How Has This Been Tested?
- squashing: lots and lots of iterations of makemigrations, migrate, and squashmigrations on both empty and loaded databases.
- management command:
  - I added some basic unit tests.
  - I also manually ran the command in my local environment, and it generated emails.

## Screenshots of your changes (if appropriate):
N/A

## Types of changes
What types of changes does your code introduce? Add an `x` in all the boxes that apply:
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: TWLight/emails/templates/emails/project_page_2021_email-body-text.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{% comment %} English-only, do not translate.{% endcomment %}
```
Comment (suecarmol) [inline TWLight/emails/templates/emails/project_page_2021_email-body-text.html:1]: Is this template used anywhere? I see that you don't import it in the `TWLight/emails/templates/emails/project_page_2021_email-body-html.html` file. Can we remove it?

File: TWLight/emails/templates/emails/project_page_2021_email-subject.html [hunk 2]
```diff
@@ -0,0 +1,2 @@
+{% comment %} English-only, do not translate.{% endcomment %}
```
Comment (suecarmol) [inline TWLight/emails/templates/emails/project_page_2021_email-subject.html:1]: Same here

**Review by suecarmol — COMMENTED** [review]
This is a great job, Jason! I know squashing migrations and solving circular dependencies can be a pain, so thanks for this PR! I just have a couple of questions before we can go ahead and merge this.

File: TWLight/emails/templates/emails/project_page_2021_email-body-text.html (hunk 1 above)
Comment (jsnshrmn) [reply TWLight/emails/templates/emails/project_page_2021_email-body-text.html:1]: This gets loaded via [djmail's templatemail](https://bameda.github.io/djmail/#_templatemail)

File: TWLight/emails/templates/emails/project_page_2021_email-subject.html (hunk 2 above)
Comment (jsnshrmn) [reply TWLight/emails/templates/emails/project_page_2021_email-subject.html:1]: This gets loaded via [djmail's templatemail](https://bameda.github.io/djmail/#_templatemail)

**Pull request merged**